### DISCUSSION TITLE
Crazyflie flips after launching the hovering scripts

### DISCUSSION THREAD

**Discussion by WoodyloveBo** [USC-ACTLab/crazyswarm#786] [Q&A]
Hi.

I am researching using multiple Crazyflies using the Crazyswarm API.

Tracking: Optitrack Camera operates at default 300FPS. The number of cameras is 4.
Area 2 x 2 x 2m
It is used in a space where there is no separate external frequency blocking. (If there is a way to check the frequency other than the radio frequency, please let me know.)

Problem: When the hoverswarm script runs, it says "Latency is high!" The phrase appears. Currently, 2 Crazyradios are connected to 4 Crazyflies. When you run the script, the drone takes off, but over time, the drone loses its pose and begins to land.

I was told this was a recalibration issue and I recalibrated it several times. But this wasn't a problem. What should I do additionally?

![Rostopic Graph](https://github.com/USC-ACTLab/crazyswarm/assets/143582301/2b5d0ab5-ee36-4d7e-aa7e-c8097fb4a45b)

This is ros_graph when crazyflie and rviz are connected. Are there any errors in detection with Natnet?

https://github.com/USC-ACTLab/crazyswarm/assets/143582301/82806929-6842-4918-b2ac-27399e766b92

https://github.com/USC-ACTLab/crazyswarm/assets/143582301/a96bab30-1d02-499f-8d1a-1b3712ee62ba

This is a video of Crazyflies hovering.

I'm having trouble progressing because of this. Please tell me how.

**Comment by whoenig** [comment]
Reduce the OptiTrack frequency to at most 100 Hz, otherwise the radio's won't be able to keep up with the incoming data.
